helm/gateway-api-config/values.yaml

```diff
@@ -1,8 +1,27 @@
-gateway:
-  name: default
-  class: default
-  host: test.gaws.gigantic.io
+baseDomain: gs.com
 
 gatewayClass:
   - name: giantswarm-default
     overrideExternalDnsHostname: ""
+
+gateway:
```

Comment:
@ubergesundheit should I use plural here? (same question for gatewayClass I guess)

Reply:
Yeah, makes sense!